src/components/Text/Text.tsx

```diff
@@ -3,10 +3,30 @@ import * as React from 'react'
 
 import { childrenExist, customPropTypes, UIComponent } from '../../lib'
 
+import { Extendable } from '../../../types/utils'
+import { ComponentVariablesInput, IComponentPartStylesInput } from '../../../types/theme'
+
+export interface ITextProps {
+  as: any
+  atMention: boolean
+  className: string
+  content: any
+  disabled: boolean
+  error: boolean
+  important: boolean
+  size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2x' | '3x' | '4x'
+  weight: 'light' | 'semilight' | 'regular' | 'semibold' | 'bold'
+  success: boolean
+  timestamp: boolean
+  truncated: boolean
+  styles: IComponentPartStylesInput
+  variables: ComponentVariablesInput
```

Comment:
@kuzhelov shouldn't all these be optional?

Reply:
yes, exactly - updating..